src/main/resources/db/migration/V5__add_tag.sql

```diff
@@ -0,0 +1,19 @@
+CREATE TABLE tags (
+  uuid        UUID PRIMARY KEY,
+  created_at  TIMESTAMP NOT NULL DEFAULT CURRENT_TIMESTAMP,
+  updated_at  TIMESTAMP NOT NULL DEFAULT CURRENT_TIMESTAMP,
+  name        VARCHAR(255) NOT NULL UNIQUE,
+  description TEXT
+);
+CREATE TABLE dataset_tag_mapping (
```

Comment:
_Minor_: For readability, let's be consistent with how we format (spacing, etc) our SQL statements :) see, [`V1__initial_schema.sql`](https://github.com/MarquezProject/marquez/blob/master/src/main/resources/db/migration/V1__initial_schema.sql#L1)

Reply:
fixed